newrelic/resource_newrelic_alert_muting_rule.go

```diff
@@ -1,13 +1,77 @@
 package newrelic
 
 import (
-	"log"
-
+	"fmt"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 	"github.com/newrelic/newrelic-client-go/pkg/errors"
+	"log"
+	"time"
 )
 
+func validateNaiveDateTime(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
Since we don't use `warns` anywhere, should this be a `_`?

Reply:
That's a good question, I'm not sure! It's part of a named return, so I don't know what the convention is for that if you don't care about one of the values. I'll poke around and see what to do about it